File: src/lib-common/is-lit-char.js
function is_lit_char(x)
{
 if(!is_str(x))
  return false
  
 if(!match_l(x,"'"))
  return false

 if(!match_r(x,"'"))
  return false
  
 let s=strip_l(x,"'")
 
 s=strip_r(s1,"'")

 if(is_empty(s))
  return false

 s=concat("\"",s,"\"")

 return is_lit(s)
} 

File: src/lib-common/is/is-lit-char.cs
fn is_lit_char x
 if not is_str x
  ret false

 if not match_l x "'"
  ret false

 if not match_r x "'"
  ret false

 let s strip_l x "'"
 let s strip_r s "'"

 if is_empty s
  ret false

 let s concat quote s

 ret is_lit s
end
